enclave/core/arena.c

```diff
@@ -35,19 +39,20 @@ void* oe_arena_malloc(size_t size)
     oe_result_t result = OE_UNEXPECTED;
     size_t total_size = 0;
     const size_t align = OE_EDGER8R_BUFFER_ALIGNMENT;
+    oe_shared_memory_arena_t* arena = _get_arena();
 
     // Create the anera if it hasn't been created.
-    if (_arena.buffer == NULL)
```

Comment:
Where do we initialize `arena->buffer` to NULL?

Reply:
Thank you Xuejun. I added an oe_once function to zero-initialize the arena.